### PR TITLE
[SPARK-20518][CORE]Supplement the new blockidsuite unit tests

### PR DESCRIPTION
**Pull request by heary-cao**
## What changes were proposed in this pull request?

This PR adds the new unit tests to support ShuffleDataBlockId , ShuffleIndexBlockId , TempShuffleBlockId , TempLocalBlockId

## How was this patch tested?

The new unit test.

### CODE REVIEW

File: core/src/test/scala/org/apache/spark/storage/BlockIdSuite.scala [hunk 1]
```diff
@@ -19,6 +19,8 @@ package org.apache.spark.storage
 
 import org.apache.spark.SparkFunSuite
 
+import java.util.UUID
```
Comment (jerryshao) [inline core/src/test/scala/org/apache/spark/storage/BlockIdSuite.scala]: The import ordering is not correct, it should be:

```
java packages

scala packages

third-party packages

spark packages
```

File: core/src/test/scala/org/apache/spark/storage/BlockIdSuite.scala [hunk 2]
```diff
@@ -101,6 +129,30 @@ class BlockIdSuite extends SparkFunSuite {
     assertSame(id, BlockId(id.toString))
   }
 
+  test("temp local") {
+    val id = TempLocalBlockId(new UUID(5,2))
```
Comment (jerryshao) [inline core/src/test/scala/org/apache/spark/storage/BlockIdSuite.scala]: nit: space after comma.

**Comment by jerryshao** [comment]
Would you please update the title to add `[core]` like other PRs.

**Comment by heary-cao** [comment]
@jerryshao 
I  have been updated it.
please review it again.
thanks.

**Comment by jerryshao** [comment]
The change LGTM, I think test PRs are always welcome. CC @srowen to see committer's comment.

**Review by srowen — APPROVED** [review]

**Comment by gatorsmile** [comment]
retest this please

**Review by srowen — APPROVED** [review]

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**